deploy/deploy_to_bintray.sh

```diff
@@ -111,6 +111,15 @@ data="{
 set +x
 ${CURL} -H Content-Type:application/json -X POST -d "${data}" "${API}/packages/${BINTRAY_REPO_OWNER}/${BINTRAY_REPO}/${PKG_NAME}/versions"
 
+echo "Adding attributes to version ${VERSION}"
+# this can be used to cleanup old versions
+pr="${TRAVIS_PULL_REQUEST:-0}"
+created=$(date -Is)
+data="[{\"name\": \"PR\", \"values\": [\"${pr}\"], \"type\": \"string\"},
+            {\"name\": \"create_time\", \"values\": [\"${created}\"], \"type\": \"date\"}]"
+echo $data
```

Comment:
I think this was added for debug purposes and should be removed

Reply:
Ok, but this is the only information that is shown this way in this script and it makes duplicate output in log:

```
Adding attributes to version PR2852_2018-11-26_07111b32
[{"name": "PR", "values": ["2852"], "type": "string"}, {"name": "create_time", "values": ["2018-11-26T20:15:52+0000"], "type": "date"}]
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   271    0   124  100   147    472    560 --:--:-- --:--:-- --:--:--   561
[{"name":"create_time","type":"date","values":["2018-11-26T20:15:52.000Z"]},{"name":"PR","type":"string","values":["2852"]}]
```

So personally I think that it is not required here

Reply:
I started to output the request data in the API call before the call for the cleanup script thus I also used it for this little snippet. This way it's easier to see what information was send and you also see the response. In this particular case the API responds with the values it recognized (they do type recognition if you omit the type parameter). In the other cases you see other response messages.

In my testing I had problems to get the json payload correct. So in case this breaks I don't want to add the output as part of the investigation but already see what the request was and develop a fix.

I probably add this kind of output to the other API calls in this script to be consistent once I polished the cleanup script.